recipes/pyomeca/meta.yaml

```diff
@@ -36,46 +35,54 @@ requirements:
   build:
     # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
     # Compilers are named 'c', 'cxx' and 'fortran'.
-    - {{ compiler('c') }}
+    # - {{ compiler('c') }}
+    - setuptools
+    - pyyaml
   host:
     - python
     - pip
   run:
     - python
+    - numpy
+    - pandas
+    - matplotlib
+    - scipy
+    - ezc3d
 
 test:
   # Some packages might need a `test/commands` key to check CLI.
   # List all the packages/modules that `run_test.py` imports.
-  imports:
-    - simplejson
-    - simplejson.tests
+  source_files:
+    - tests
+  requires:
+    - pytest
+    - pytest-cov
+  commands:
+    - pytest tests
 
 about:
-  home: http://github.com/simplejson/simplejson
+  home: http://github.com/{{ name }}/{{ name }}
   # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
   # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
   # See https://opensource.org/licenses/alphabetical
-  license: MIT
+  license: Apache-2.0
   # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
-  license_family: MIT
+  license_family: Apache
   # It is strongly encouraged to include a license file in the package,
   # (even if the license doesn't require it) using the license_file entry.
   # See http://conda.pydata.org/docs/building/meta-yaml.html#license-file
-  license_file: LICENSE.txt
-  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+  license_file: LICENSE
+  summary: 'Pyomeca is a python library allowing to carry out a complete biomechanical analysis; in a simple, logical and concise way'
 
   # The remaining entries in this section are optional, but recommended.
   description: |
-    simplejson is a simple, fast, complete, correct and extensible
-    JSON <http://json.org> encoder and decoder for Python 2.5+ and
-    Python 3.3+. It is pure Python code with no dependencies, but includes
-    an optional C extension for a serious speed boost.
-  doc_url: http://simplejson.readthedocs.io/
-  dev_url: https://github.com/simplejson/simplejson
+    Python toolbox for biomechanics analysis
+  doc_url: https://github.com/{{ name }}/{{ name }}
+  dev_url: https://github.com/{{ name }}/{{ name }}
 
 extra:
   recipe-maintainers:
     # GitHub IDs for maintainers of the recipe.
     # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
-    - LisaSimpson
-    - LandoCalrissian
+    - mrtnz
```

Comment:
@mrtnz Are you happy to be listed as a maintainer of this recipe?

Reply:
Yes, thanks.